sdk/test/UnitTests/Custom/Marshalling/BackwardsCompatibilityForDateTimePropertiesTests.cs

```diff
@@ -36,91 +36,22 @@ public class BackwardsCompatibilityForDateTimePropertiesTests
-            var marshalledRequest = marshaller.Marshall(request);
-
-            Assert.AreEqual(marshalledRequest.Parameters["StartTime"], EXPECTED_MARSHALLED_STRING);
-            Assert.AreEqual(marshalledRequest.Parameters["EndTime"], EXPECTED_MARSHALLED_STRING);
-            Assert.AreEqual(marshalledRequest.Parameters["Time"], EXPECTED_MARSHALLED_STRING);
-        }
-
-        [TestMethod]
-        [TestCategory("UnitTest")]
-        public void TestLegacyFieldsBackwardsCompatibilityWithMS()
-        {
-            PutScheduledUpdateGroupActionRequest request = new PutScheduledUpdateGroupActionRequest();
-
-            DateTime timeUtc = new DateTime(2018, 7, 18, 0, 0, 0, 123, DateTimeKind.Utc);
-            DateTime timeLocal = new DateTime(2018, 7, 18, 0, 0, 0, 123, DateTimeKind.Local);
-            DateTime timeUnspecified = new DateTime(2018, 7, 18, 0, 0, 0, 123);
+            //This is a value use of .ToLocalTime to construct a local time for testing.
+            DateTime timeLocal = new DateTime(2018, 7, 18, 0, 0, 0, DateTimeKind.Utc).ToLocalTime();
+            DateTime timeUnspecified = new DateTime(timeLocal.Ticks);
 
-#pragma warning disable CS0618 // Type or member is obsolete
             request.StartTime = timeUtc;
             request.EndTime = timeLocal;
             request.Time = timeUnspecified;
-
-            Assert.AreEqual(request.StartTime, timeUtc);
-            Assert.AreEqual(request.EndTime, timeLocal);
-            Assert.AreEqual(request.Time, timeUnspecified);
-#pragma warning restore CS0618 // Type or member is obsolete
-
-            Assert.AreEqual(request.StartTimeUtc, timeUtc);
-            Assert.AreEqual(request.EndTimeUtc, timeUtc);
-            Assert.AreEqual(request.TimeUtc, timeUtc);
-
-            var marshaller = new PutScheduledUpdateGroupActionRequestMarshaller();
-            var marshalledRequest = marshaller.Marshall(request);
-
-            Assert.AreEqual(marshalledRequest.Parameters["StartTime"], EXPECTED_MARSHALLED_STRING_WITH_MS);
-            Assert.AreEqual(marshalledRequest.Parameters["EndTime"], EXPECTED_MARSHALLED_STRING_WITH_MS);
-            Assert.AreEqual(marshalledRequest.Parameters["Time"], EXPECTED_MARSHALLED_STRING_WITH_MS);
-        }
-
-        [TestMethod]
-        [TestCategory("UnitTest")]
-        public void TestUtcFields()
-        {
-            PutScheduledUpdateGroupActionRequest request = new PutScheduledUpdateGroupActionRequest();
-
-            DateTime timeUtc = new DateTime(2018, 7, 18, 0, 0, 0, DateTimeKind.Utc);
-            DateTime timeLocal = new DateTime(2018, 7, 18, 0, 0, 0, DateTimeKind.Utc).ToLocalTime();
-            DateTime timeUnspecified = new DateTime(timeLocal.Ticks);
-
-            request.StartTimeUtc = timeUtc;
-            request.EndTimeUtc = timeLocal;
-            request.TimeUtc = timeUnspecified;
-
-#pragma warning disable CS0618 // Type or member is obsolete
             Assert.AreEqual(request.StartTime, timeUtc);
```

Comment:
It could if we broke the code and changed the property StartTime to convert to localtime for example on `set`. So it is just verifying the value set it not touched be it UTC, Local, or Unspecified. No conversions should be done at this time.